src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -72,3 +76,70 @@ def get(self, request: Request, organization: Organization) -> Response:
             order_by="position",
             on_results=lambda x: serialize(x, request.user, serializer=GroupSearchViewSerializer()),
         )
+
+    def put(self, request: Request, organization: Organization) -> Response:
```

Comment:
are we updating multiple `GroupView`s in this put?
why PUT instead of POST then?

Reply:
This PUT endpoint is designed to reduce complexity on the frontend by allowing it to just send over the list of views it sees over to the backend. The backend takes this list of `GroupSearchView`s, and handles the logic for adding, updating, and deleting to the database in bulk. We'll consider adding public facing endpoints that adhere to RESTful conventions (like a POST endpoint that updates one view) in the future once we get closer to launching this feature.

This should also address this comment below:
> hmm should the deletions be done in a separate endpoint?